app/javascript/sdk/IFrameHelper.js

```diff
@@ -166,6 +175,9 @@ export const IFrameHelper = {
     iframe.style.visibility = '';
     iframe.setAttribute('id', `chatwoot_live_chat_widget`);
 
+    const closeBtn = document.getElementById('closeBtn');
+    closeBtn.style.visibility = '';
+
```

Comment:
button is initially hidden and we make it visible here when iframe becomes visible.